common/app/implicits/Requests.scala

```diff
@@ -32,7 +32,16 @@ trait Requests {
 
     lazy val isHealthcheck: Boolean = r.headers.keys.exists(_ equalsIgnoreCase  "X-Gu-Management-Healthcheck")
 
-    def isInImgixTest: Boolean = Switches.ImgixSwitch.isSwitchedOn && (imgixTestSections.exists(r.path.startsWith) || getParameter("inImgixTest").nonEmpty)
+
+    private val imgixTestSections = Seq(
+      "/uk/money",
+      "/au/money",
+      "/us/money",
+      "/money"
+    )
+
+    lazy val isInImgixTest: Boolean = Switches.ImgixSwitch.isSwitchedOn &&
+      (Configuration.environment.isNonProd || imgixTestSections.exists(r.path.startsWith))
```

Comment:
Slightly more subtle than that.

The switch is binary on/ off - this means it is limited to whitelisted sections in PROD but on in all sections elsewhere.